android/app/src/main/java/com/reactnativesampleapp/OnfidoSDK.java

```diff
@@ -42,6 +43,11 @@ public void userCompleted(Applicant applicant, Captures captures) {
                 public void userExited(ExitCode exitCode, Applicant applicant) {
                     mErrorCallback.invoke(exitCode.toString());
                 }
+
+                @Override
+                public void onError(OnfidoException e, Applicant applicant) {
+                    mErrorCallback.invoke("Onfido SDK errored");
```

Comment:
Can we replace `"Onfido SDK errored"` by `e.getMessage()`? It gives the integrator more context on what happened